lib/SocketManager.js

```diff
@@ -15,6 +15,7 @@ const socketEvents = {
   nodeUpdated: 'NODE_UPDATED',
   valueUpdated: 'VALUE_UPDATED',
   valueRemoved: 'VALUE_REMOVED',
+  valueWritten: 'VALUE_WRITTEN',
```

Comment:
Not here. Here: https://github.com/zwave-js/zwavejs2mqtt/blob/master/types/index.d.ts#L323